workers/unity/Assets/Fps/Scripts/SetupLogic/SimulatedPlayerWorkerConnector.cs

```diff
@@ -38,4 +74,39 @@ protected override ReceptionistConfig GetReceptionistConfig(string workerType)
 
         return config;
     }
+
+    protected override AlphaLocatorConfig GetAlphaLocatorConfig(string workerType)
+    {
+        var pit = GetDevelopmentPlayerIdentityToken(simulatedPlayerDevAuthTokenId, "SimulatedPlayer", GetDisplayName());
+        var loginTokens = GetDevelopmentLoginTokens(workerType, pit);
+        var loginToken = SelectLoginToken(loginTokens);
+
+        return new AlphaLocatorConfig
+        {
+            LocatorHost = RuntimeConfigDefaults.LocatorHost,
+            LocatorParameters = new LocatorParameters
+            {
+                PlayerIdentity = new PlayerIdentityCredentials
+                {
+                    LoginToken = loginToken,
+                    PlayerIdentityToken = pit
+                }
+            }
+        };
+    }
+
+    protected override string SelectLoginToken(List<LoginTokenDetails> loginTokens)
+    {
+        var selectedLoginToken = loginTokens.First(token => token.DeploymentName == simulatedPlayerTargetDeployment)
+            .LoginToken;
```

Comment:
wouldn't this already throw a null pointer exception when it can't find a matching token?

Reply:
Fixed!